src/improvements/template/FinanceTemplate.sol

```diff
@@ -77,6 +77,9 @@ contract FinanceTemplate is SimpleBase {
     /// @notice Sets up the template with module configuration from a TOML file
     /// @param taskConfigFilePath Path to the TOML configuration file
     function _templateSetup(string memory taskConfigFilePath) internal override {
+        bytes memory multicall3NoCheckBytecode = vm.getDeployedCode("Multicall3NoCheck");
+        // TDOD: Remove this once Multicall3NoCheck is deployed
+        vm.etch(address(multicallTarget), multicall3NoCheckBytecode);
```

Comment:
Let's block this PR on deployment so we don't forget this, once the related comments above for this contract are addressed we (OP Labs) can deploy